src/bin/dist.ts

```diff
@@ -1,23 +1,49 @@
 #!/usr/bin/env node
 
+import * as _ from "lodash";
 import config from "../config/config";
 import {ScriptLinter} from "../scripts";
 import {StyleCompiler, StyleLinter} from "../styles";
 import {TranslationCompiler} from "../translations";
 import {Assets} from "../assets";
+import {StylesCompileConfig} from "../config/config.interface";
 
-const onError = () =>
-  process.exit(1);
+const onError = () => process.exit(1);
 
-const assets = new Assets(config.assets.dist).on("error", onError);
+let assets: Assets;
 
-assets
-  .clean()
-  .on("clean", () => {
+const startTooling = () => {
+  if (assets) {
     assets.copy();
+  }
+  if (config.scripts) {
     new ScriptLinter(config.scripts.lint.dist).on("error", onError).start();
-    new StyleLinter(config.styles.lint.dist).on("error", onError).start();
-    config.styles.dist.forEach(styleConfig =>
-      new StyleCompiler(styleConfig).on("error", onError).start());
+  }
```

Comment:
We have to wait for the bundle. And the bundling is not handled by the tooling. We could think about using the waitFor flag like in the dev mode.